docs/sources/panels-visualizations/visualizations/time-series/index.md

```diff
@@ -178,11 +178,15 @@ The following example shows three series: Min, Max, and Value. The Min and Max s
 
 Options under the axis category change how the X and Y axes are rendered. Some options do not take effect until you click outside of the field option box you are editing. You can also or press `Enter`.
```

Comment:
@imatwawana - not sure how we want to handle capitalization of x / y axis in the this case 🤔 maybe still just uncapitalize them always? i.e. `x and y axes` vs `X and Y axes`

Reply:
yes 100% please take those capitals away. and it should really be `x- and y-axes`.

Reply:
got changed it and fixed last remaining `Y-axes` that I missed from before (just searching `y-axis` 😬

Reply:
thank you! much appreciated!